docs/learn-unique-features.md

```diff
@@ -5,6 +5,11 @@ sidebar_label: Unique features
 slug: /unique-features
 ---
 
-- The `ldpos-chain` allows you to change passphrases. A transaction is needed for it to go in effect.
+- The `ldpos-chain` allows you to change passphrases. A transaction is needed for it take effect.
```

Comment:
Should be "for it to take effect" ("to" is missing)

Reply:
See 6af22f505e71d5d1b388e7b78a9601fbb641cfb1.